cmake/catkin_install_python.cmake

```diff
@@ -35,26 +35,47 @@ function(catkin_install_python signature)
         # write modified file with modified shebang line
         get_filename_component(python_name ${PYTHON_EXECUTABLE} NAME)
         string(REGEX REPLACE "${regex}" "#!/\\1/env ${python_name}\\2" data "${data}")
-        get_filename_component(filename ${file} NAME)
-        set(file "${CMAKE_CURRENT_BINARY_DIR}/catkin_generated/installspace")
-        file(MAKE_DIRECTORY ${file})
-        set(file "${file}/${filename}")
-        file(WRITE ${file} "${data}")
+        get_filename_component(filename ${source_file} NAME)
+        set(rewritten_file "${CMAKE_CURRENT_BINARY_DIR}/catkin_generated/installspace")
+        file(MAKE_DIRECTORY ${rewritten_file})
+        set(rewritten_file "${rewritten_file}/${filename}")
+        file(WRITE ${rewritten_file} "${data}")
       endif()
       # install (modified) file to destination
       set(optional_flag "")
       if(ARG_OPTIONAL)
         set(optional_flag "OPTIONAL")
       endif()
-      install(PROGRAMS "${file}" DESTINATION "${ARG_DESTINATION}" ${optional_flag})
+      # Install copy of file with re-written shebang to install space
+      install(PROGRAMS "${rewritten_file}" DESTINATION "${ARG_DESTINATION}" ${optional_flag})
```

Comment:
`rewritten_file` isn't defined if the above code didn't enter the `if` block.

Reply:
Fixed in f9e2e6b